controller/src/main/java/org/jboss/as/controller/CapabilityReferenceRecorder.java

```diff
@@ -47,6 +47,10 @@
      */
     void removeCapabilityRequirements(OperationContext context, String attributeName, String... attributeValues);
 
+    String getBaseDependentName();
+
+    String getBaseRequirementName();
+
```

Comment:
You might as well expose isDynamicDependent() as well. I believe Elytron branch has a custom impl of this interface, which this will break, so we might as well get everything in and only break them once. ;)